test/jbuild

```diff
@@ -2,7 +2,7 @@
 
 (executables
  ((names (test_dblib test_ct))
-  (libraries (freetds oUnit))))
+  (libraries (freetds oUnit threads))))
```

Comment:
I think `threads` must be first — and maybe an additional command option `-thread` must be added.

Reply:
This worked?